jwt/algorithms.py

```diff
@@ -534,9 +538,9 @@ def verify(self, msg, key, sig):
             except InvalidSignature:
                 return False
 
-    class Ed25519Algorithm(Algorithm):
+    class OKPAlgorithm(Algorithm):
```

Comment:
I think we might need to consider this a breaking change.

Reply:
Thanks for your time reviewing this PR.

I think Algorithm classes are not disclosed via jwt/__init__.py so it is not necessary for us to regard this change as a breaking change.

As you can see, there is no change on public class tests (e.g., jwt_test.py, jws_test.py, jwk_test.py) except for [here](https://github.com/jpadilla/pyjwt/pull/675/files#diff-a18f5631d08da144d86418dfbc5b72725aa3801d9eda3d0a38df7acc72e1b8ccR164) where a test assertion checks an instance type of `jwk.Algorithm` returned.

Reply:
@auvipy Hi, could you review and merge it?

Reply:
my only  concern is  what happen to Ed25519Algorithm? can we use deprecation warning to it and introduce  OKPAlgorithm at the same time?

Reply:
> can we use deprecation warning to it and introduce OKPAlgorithm at the same time?

We don't need to use deprecation warning because `Ed25519Algorithm` is a private class and this change does not break backward-compat. All of the tests for public classes and methods are passed successfully.